warp10/src/main/java/io/warp10/continuum/Configuration.java

```diff
@@ -672,6 +677,11 @@
    */
   public static final String INGRESS_PORT = "ingress.port";
   
+  /**
+   * Port onto which the ingress server should listen
+   */
+  public static final String INGRESS_TCP_BACKLOG = "ingress.tcp.backlog";
```

Comment:
Needs to be split in something like ingress.http.backlog and ingress.ssl.backlog to add the same logic to the SSL endpoint.